data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,472 @@ mission "Unfettered returning home"
+			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us, but maybe someday you will prove yourself otherwise."`
+	to fail
+		not "flagship landed"
+		has "unfettered: fled with the relic"
+	on fail
+		"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+		clear "old reputation: Hai (Unfettered)"
+		dialog `The Unfettered hail you. "You failed to complete the duel, and as such, are no longer authorized to land on our worlds. You will not enjoy unwarranted respect any longer! It would be in your best interest to return the ship to us on Darkcloak, but any of our worlds will do."`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+		personality unconstrained launching plunders staying target heroic disables marked
+		ship "Lightning Bug (Ionic)" "Eraser"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		on kill
+			dialog `When you destroy the last dueling ship, you receive a hail from Mera, who sounds quite angry. "Destroying one or two ships in the middle of the action could be understood. But all of them? That's either incompetence or evil intent, and I will not have either."`
+				fail
+		conversation
+			`You receive a message from the remaining Unfettered ships: "We surrender. If you land now, you will receive your reward, so long as you do not capture our ships."`
+			branch "good or bad enough"
+				or
+					has "unfettered: beaten extra bugs"
+					"old reputation: Hai (Unfettered)" >= -2000
+			`	"Do not declare victory too soon, however, for we were not the only ones you had to beat," another adds.`
+			label "good or bad enough"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+			"old reputation: Hai (Unfettered)" < -2000
+		personality unconstrained entering plunders target heroic mute marked
+		ship "Sea Scorpion (Shredder)" "Requiem"
+		system "Hi Yahr"
+		conversation
+			branch "first one"
+				not "unfettered: beaten extra bugs"
+			`You have managed to defeat the last of the Unfettered challengers. Now it's time to land and claim your reward.`
+				goto score
+			label "first one"
+			`You receive no hail from this extra challenger, but there can be no doubt that they are indeed defeated.`
+			label score
+			action
+				set "unfettered: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			not "unfettered: fled with the relic"
+		ship "Lightning Bug (Unfettered Shipyards)"
```

Comment:
This one needs a name.

![image](https://github.com/endless-sky/endless-sky/assets/18634983/9a22ae3d-ca79-40b6-9653-d5dc2d7b808f)

Reply:
```suggestion
		ship "Lightning Bug (Unfettered Shipyards)" "Eye in the Sky"
```
Maybe this given its not a warlord ship
Or Dungeon Master or smth like that?

Reply:
[probably a bad idea] if you want to be mysterious, you could do;
```suggestion
		ship "Lightning Bug (Unfettered Shipyards)" "Oubliette Ruler"
```
Both the downside and the upside is that very few people will know what it means, and it will sound strange.